File: programme/count-capital-letters/countCapitalLetters.js
// Write a programme to count capital letters in a string

function Capital(str) {
  let count = 0;
  for (let i = 0; i <= str.length - 1; i++) {
    if (str[i] === str[i].toUpperCase()) {
      count += 1;
    }
  }
  return count;
}

const str = "HiiThere";
console.log("Input  : " + str);
console.log("Output : " + Capital(str));

File: programme/count-capital-letters/count-capital-letters.jl
# Write a programme to count capital letters in a string

function capitals()
  counter = 0
  print("Input  : ")
  input = readline()
  for character in input
    if all(isuppercase, character)
      counter = counter + 1
    end
  end
  println("Output : ", counter)
end

capitals()
